_examples/main.go

```diff
@@ -2,11 +2,15 @@ package main
 
 import (
 	"fmt"
-	"github.com/fluent/fluent-logger-golang/fluent"
-	// "../fluent"
+	"time"
+
+	"../fluent"
+
+	"github.com/Sirupsen/logrus"
```

Comment:
Adding external dependency only for this example doesn't make sense... 
"log" in std library may be enough.